roofit/roofitcore/inc/RooFit/ModelConfig.h

```diff
@@ -369,8 +368,7 @@ class ModelConfig final : public TNamed, public RooWorkspaceHandle {
    std::unique_ptr<RooAbsReal> createNLLImpl(RooAbsData &data, const RooLinkedList &cmdList) const;
    std::unique_ptr<RooFitResult> fitToImpl(RooAbsData &data, const RooLinkedList &cmdList) const;
 
-   ClassDefOverride(ModelConfig,
-                    6); ///< A class that holds configuration information for a model using a workspace as a store
+   ClassDefOverride(ModelConfig, 7);
```

Comment:
What is the plan to read file containing version 6?

Reply:
Automatic schema evolution: a member was made transient from version 6 to 7. The transient member will be filled now in the `RooWorkspace::Streamer()` when reading, since any ModelConfig is always stored inside a RooWorkspace, or the workspace pointer `fWSName` is not set anyway.

Reply:
Fair enough I had missed that `fRefWS` is now 'transient'.   Nonetheless we should test reading the old file to make sure there is not a spurrious message about `TRef` not being readable into a `RooWorkspace*`.

Reply:
Maybe related/affected by: https://its.cern.ch/jira/browse/ROOT-8837

Reply:
Good point. I have added a unit test for reading an old file with a ModelConfig version 6.